templates/etc/td-agent/td-agent.conf

```diff
@@ -28,10 +28,18 @@
   buffer_chunk_limit 1M
   # Flush logs every 5 seconds, even if the buffer is not full.
   flush_interval 5s
-  # Never wait longer than 5 minutes between retries.
+  # Enforce some limit on the number of retries.
+  disable_retry_limit false
+  # After 3 retries, a given chunk will be discarded.
+  retry_limit 3
+  # Wait 10 seconds before the first retry. The wait interval will be doubled on
+  # each following retry (20s, 40s...) until it hits the retry limit.
+  retry_wait 10
+  # Never wait longer than 5 minutes between retries. If the wait interval
+  # reaches this limit, the exponentiation stops. Given the default config, this
```

Comment:
Let's put this sentence (starting with "Given") on its own line, so it's more visible. Also, the next one (starting with "But") can be folded into this one.
Also in the other files.

Reply:
Done.